code-scanning/properties/sysdig-scan.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Sysdigh Inline Scan",
+    "creator": "Sysdig",
+    "description": "Performs analysis on locally built container image and posts the results in SARIF report",
+    "iconName": "sysdig",
+    "categories": ["Code Scanning", "C", "C#", "C++", "Go", "Java", "JavaScript", "Node", "Python", "Powershell", "Ruby"]
```

Comment:
since the workflow builds docker image, it is a good idea to include "Dockerfile" as a category.

Reply:
It is done, thank you for that